charmcraft.yaml

```diff
@@ -3,22 +3,7 @@ parts:
   charm:
     source: .
     plugin: reactive
-    # using charm/3.x/stable will cause the charm unable to install on 18.04
-    # with the following error: "pip requires Python '>=3.7' but the running
-    # Python is 3.6.9". This is because charm/3.x/stable start using system
-    # installation of python instead of a python snap.
     build-snaps: [charm/2.x/stable]
```

Comment:
If we're not supporting 18.04 builds any more (which we aren't according to the bases/platforms definitions below), can we drop this pin?

Reply:
it seems that didn't work (check workflow for https://github.com/canonical/charm-openstack-service-checks/pull/202/commits/38890a8d659b43941079c150e928af10ebd77815)